org.eclipse.m2e.profiles.ui/src/org/eclipse/m2e/profiles/ui/internal/actions/ProfileSelectionHandler.java

```diff
@@ -72,9 +78,19 @@ public class ProfileSelectionHandler extends AbstractHandler {
 
   private static final Logger log = LoggerFactory.getLogger(ProfileSelectionHandler.class);
 
+  @Inject
+  private IProfileManager profileManager;
+
+  public ProfileSelectionHandler() {
+    BundleContext context = FrameworkUtil.getBundle(ProfileSelectionHandler.class).getBundleContext();
+    IEclipseContext serviceContext = EclipseContextFactory.getServiceContext(context);
```

Comment:
This plug-ins manifest specifies `Bundle-ActivationPolicy: lazy` so the `BundleContext` should never be null here.
Furthermore I think this code will never run outside of OSGi so I also expect that `getBundle()` never returns null unless the entire framework is in a weird state.